buffer/src/test/java/io/netty/buffer/NoopResourceLeakTracker.java

```diff
@@ -15,22 +15,27 @@
  */
 package io.netty.buffer;
 
-import io.netty.util.ResourceLeak;
+import io.netty.util.ResourceLeakTracker;
 
-final class NoopResourceLeak implements ResourceLeak {
+import java.util.concurrent.atomic.AtomicBoolean;
 
-    static final NoopResourceLeak INSTANCE = new NoopResourceLeak();
 
-    private NoopResourceLeak() { }
+final class NoopResourceLeakTracker extends AtomicBoolean implements ResourceLeakTracker<ByteBuf> {
```

Comment:
rename to `NoopByteBufResourceLeakTracker` or just make this class generic (preferably the latter)